src/installer/tests/TestUtils/Command.cs

```diff
@@ -196,7 +197,22 @@ public Command Start()
 
             ReportExecBegin();
 
-            Process.Start();
+            // Retry if we hit ETXTBSY due to Linux race
+            // https://github.com/dotnet/runtime/issues/58964
+            for (int i = 0; i < 3; i++)
```

Comment:
just make this without `i < 3` and move the exit condition to the filter. - The exit should be either success or a crash.